### DISCUSSION TITLE
Index not works.

### DISCUSSION THREAD

**Discussion by andy263** [milvus-io/milvus#16019] [Q&A and General discussion]
Milvus:v2.0.0-rc8-20211104-d1f4106
PyMilvus:2.0.0rc7

Vectors 2M, Dim 2048, querynode 8c32g*4, indexnode 8c32g*4, datanode 8c32g*4

nprobe 4  top_k 10

With or without any type of indices, search costs approximately 500ms. Why?

**Comment by xiaojuan8** [comment]
fyi: https://github.com/milvus-io/milvus/discussions/15689
or wait for maintainer comes to answer.

**Comment by xiaofan-luan** [comment]
> 

Hi andy, would better recommend you to use 2.0.1 for testing, don't forget to upgrade the pymilvus version too.

The 2.0.0-RC7 use strong consistency as default so it will take much longer on waiting for data
